index.html

```diff
@@ -191,9 +201,20 @@
 </header>
 
     <div id="box">
-        <form>
+        <noscript>
+            <div>
+                <h2><b>Say hello 👋 to ...</b> you've disabled JavaScript, no problem:</h2>
```

Comment:
maybe discard the "say hello to ..." ?

Reply:
the idea is that the user can get an idea about what the following steps are about directly in the headline; therefore i did not start with "JavaScript disabled"